cdap-ui/app/tracker/controllers/dictionary-ctrl.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+class TrackerDictionaryController {
+  constructor($state, $scope, myTrackerApi, $uibModal) {
+    this.$state = $state;
+    this.$scope = $scope;
+    this.myTrackerApi = myTrackerApi;
+    this.$uibModal = $uibModal;
+    this.currentPreferredPage = 1;
+    this.currentUserPage = 1;
+    this.addNewColumn = false;
+    this.dictionary = {
+      dictionaryData: []
```

Comment:
Is there any other property to `this.dictionary`? It doesn't seem like it, and if so, is there any specific reason why the dictionaryData array is within another object?